src/library/Uno.Material/Styles/Controls/v2/NavigationView.xaml

```diff
@@ -208,7 +208,7 @@
 
             <StaticResource x:Key="NavigationViewItemRippleFeedback" ResourceKey="PrimaryPressedBrush" />
 
-            <StaticResource x:Key="TopNavigationViewItemForeground" ResourceKey="SystemControlTransparentBrush" />
```

Comment:
We'll need to update the resource docs for all changes resource values:

https://github.com/unoplatform/Uno.Themes/blob/master/doc/styles/NavigationView.md#lightweight-styling